Flabbergasted-Flamingos/bangazon-api
# Rb doc strings

# Description
I added very simple descriptions for each model, view and serializer that included author name and what the class represents

## Number of Fixes
fixes on all models

## Related Ticket(s)
affects #2 #3 #4 #5 #6 #7 #8 #10 

## Problem to Solve
Helps developers know who wrote what

## Proposed Changes
Affects 
api/serializer.py
api/models.py
api/views.py

## Expected Behavior
Does not affect behaviour, this is documentation

## Steps to Test Solution

git fetch --all
git checkout rb-docStrings

Have a look around, if you see any incorrect authorship please let me know!
